### PR TITLE
Fixed Resource class

### PR DESCRIPTION
**Pull request by lyrixx**

### CODE REVIEW

File: src/Client/Resource.php [hunk 1]
```diff
@@ -15,9 +13,9 @@
     const FETCH_PROMISE = 'promise';
 
     /**
-     * @var HttpClient|HttpAsyncClient
+     * @var FlexibleHttpClient
      */
-    protected $httpAsyncClient;
+    protected httpClient;
```
Comment (welcoMattic) [inline src/Client/Resource.php]: `$` is missing 👁

Comment (ternel) [reply src/Client/Resource.php]: MOAR 💰  plzzzz

Comment (lyrixx) [reply src/Client/Resource.php]: lol, oups. Edit in place in github is not really the way to go!

Comment (lyrixx) [reply src/Client/Resource.php]: Fixed. Good catch.

**Comment by lyrixx** [comment]
ping @joelwurtz

**Comment by lyrixx** [comment]
ping @Korbeil

**Comment by Korbeil** [comment]
Since it's a read-only repository, you should move it to https://github.com/janephp/janephp please.

**Pull request closed without merging**